tst/EnergyPlus/unit/Coils/CoilCoolingDX.unit.cc

```diff
@@ -325,6 +326,13 @@ TEST_F(CoilCoolingDXTest, CoilCoolingDXAlternateModePerformanceHitsSaturation)
         EXPECT_NEAR(10.238, evapOutletNode.Temp, 0.01);
         EXPECT_NEAR(0.007748, evapOutletNode.HumRat, 0.0001);
     }
+    EXPECT_EQ(thisCoil.availSched->currentVal, 1.0);
+    EXPECT_EQ(thisCoil.performance->coilCoolingDXAvailSched->currentVal, 1.0);
+    auto coilPerformance{dynamic_cast<EnergyPlus::CoilCoolingDXCurveFitPerformance *>(thisCoil.performance.get())};
+    EXPECT_EQ(coilPerformance->normalMode.coilCoolingDXAvailSched->currentVal, 1.0);
+    EXPECT_EQ(coilPerformance->alternateMode.coilCoolingDXAvailSched->currentVal, 1.0);
+    EXPECT_EQ(coilPerformance->alternateMode2.coilCoolingDXAvailSched->currentVal, 1.0);
+
```

Comment:
Modified an existing unit test to show that the current value of the schedule pointer variable is correctly updated.